.github/workflows/main.yml

```diff
@@ -28,3 +28,5 @@ jobs:
         run: yarn build
       - name: Run Tests
         run: yarn test
+      - name: Run Typecheck
```

Comment:
@cstenglein fyi, added this to avoid pushing failing TS in the future

Reply:
Yes I saw that and wasn't sure about it.

But since you made it work in this branch I think we can add it in general, so we have stricter checks in the future :+1: